sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceDeleteSuccessEventData.java

```diff
@@ -223,18 +230,44 @@ public ResourceDeleteSuccessEventData setStatus(String status) {
      * Get the authorization property: The requested authorization for the operation.
      *
      * @return the authorization value.
+     * @deprecated This method is no longer supported since v4.9.0.
+     * <p> Use {@link ResourceDeleteSuccessEventData#getResourceAuthorization()} instead.
      */
+    @Deprecated
     public String getAuthorization() {
-        return this.authorization;
+        return this.authorizationString;
```

Comment:
We need to use the serializer here as well similar to what we are doing in `ResourceDeleteCancelEventData`.